DESCRIPTION

```diff
@@ -31,7 +31,8 @@ Suggests:
     knitr,
     rmarkdown,
     testthat (>= 3.0.3),
-    vdiffr (>= 1.0.0)
+    vdiffr (>= 1.0.0),
+    withr
```

Comment:
`{testthat}` already relies on `{withr}` (https://cran.r-project.org/web/packages/testthat/index.html), so technically this is not necessary. I am adding it nonetheless to avoid a `WARNING` in R CMD Check.